build-info-extractor/src/main/java/org/jfrog/build/extractor/BuildInfoExtractorUtils.java

```diff
@@ -179,36 +180,37 @@ public static String buildInfoToJsonString(BuildInfo buildInfo) throws IOExcepti
         JsonFactory jsonFactory = createJsonFactory();
 
         StringWriter writer = new StringWriter();
-        JsonGenerator jsonGenerator = jsonFactory.createJsonGenerator(writer);
-        jsonGenerator.useDefaultPrettyPrinter();
-
-        jsonGenerator.writeObject(buildInfo);
-        String result = writer.getBuffer().toString();
-        return result;
+        try (JsonGenerator jsonGenerator = jsonFactory.createGenerator(writer)) {
```

Comment:
Let's add the StringWriter to the try..with..resources

Reply:
done